pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -307,6 +314,7 @@ type CephStatus struct {
 	LastChanged    string                       `json:"lastChanged,omitempty"`
 	PreviousHealth string                       `json:"previousHealth,omitempty"`
 	Capacity       Capacity                     `json:"capacity,omitempty"`
+	Versions       CephDaemonsVersions          `json:"versions,omitempty"`
```

Comment:
Because this field is optional it should be a pointer.

Reply:
Done.